src/cmd/serve.rs

```diff
@@ -529,7 +530,12 @@ pub fn serve(
                         if path.is_dir() && is_folder_empty(&path) {
                             continue;
                         }
-                        println!("Change detected @ {}", Local::now().format("%Y-%m-%d %H:%M:%S"));
+
+                        let format = format_description!("[year]-[month]-[day] [hour]:[minute]:[second] UTC");
+                        println!(
+                            "Change detected @ {}",
+                            OffsetDateTime::now_utc().format(&format).unwrap().to_string()
+                        );
```

Comment:
@Keats This timestamp is the only reason the root crate needs a dependency on `time`.  It's also a very very minor regression because with `chrono`, Zola could print the change detected timestamp in the user's local timezone.  [`time` can't do that](https://github.com/time-rs/time/issues/427) in multithreaded applications.  There's some kind of UB/unsoundness around getting local time in a thread, which is why this is now printing UTC times.

What do you think of just doing `println!("Change detected")` and dropping the timestamp?  Then the root crate wouldn't need a dependency on time.

Reply:
Is it only on MacOS? I kind of like having the datetime in there to make sure I'm getting the changes when I expect them (eg I see nothing changing on the site and I check the the terminal to see if it has been picked up). 
At the same time, if it's not showing the local timezone it's not worth it.
It's kinda weird it's not possible though, can we get the user offset on startup before starting any threads and re-using it?

Reply:
Seems like that should be possible, I'll look into it.

Reply:
That issue indicates it is only on Mac OS but it was happening to me on Fedora. Calling now_local was Err.

Reply:
@Keats Grabbing the UTC offset before threading starts worked like a charm.  patch pushed to this branch.

Reply:
I did have to add another argument to `serve()`, but thankfully `#[allow(clippy::too_many_arguments)]
` was already there. :sweat_smile: